tests/testthat/test-unnest.R

```diff
@@ -114,7 +114,7 @@ test_that("unpacks df-cols (#1112)", {
 
 test_that("unnesting column of mixed vector / data frame input is an error", {
   df <- tibble(x = list(1, tibble(a = 1)))
-  expect_snapshot((expect_error(unnest(df, x))))
+  expect_snapshot(unnest(df, x), error = TRUE, cnd_class = TRUE)
```

Comment:
I took a look at the updated snapshots. In all cases I believe we can drop `cnd_class = TRUE`. 

Our advice lately is that if you _really_ care about the error class, then you do

```r
testthat::expect_error(class = )
```

and if you _also_ care about the error message, you do a separate

```r
testthat::expect_snapshot(error = )
```

But lately we have not been combining both features into a single snapshot

And in all cases here in tidyr I don't think we care about the error class

Reply:
Done in b49c272d93b43b90564f8303af7624077973e0bc